mscore/musescore.cpp

```diff
@@ -246,6 +252,9 @@ const std::list<const char*> MuseScore::_basicNoteInputMenuEntries {
             "",
             "pad-rest",
             "",
+            "pad-raise",
+            "pad-lower",
+            "",
```

Comment:
I think it would make MuseScore a better program for most users to have it in Basic.
There is reasoning explained in the related issue for why this feature is necessary to have a good music editing program. (This reasoning is not based at all on Finale or other such influences.)
And Finale has spent a lot of money researching which commands are most used. We can take advantage of money that someone else has already spent, to make MuseScore better.

Being able to edit the note input toolbar makes it usable for me. So if you say that you will not accept the pull request unless it is removed from Basic, then I will remove it from Basic.